modules/70_water/heat/realization.gms

```diff
@@ -6,10 +6,24 @@
 *** |  Contact: remind@pik-potsdam.de
 *** SOF ./modules/70_water/heat/realization.gms
 
-*' @description Exogenous water demand is calculated based on data on water demand coefficients and cooling shares. 
-*' Vintage structure in combination with time dependent cooling shares as vintages and efficiency factors are also considered.
-*' Demand is a function of excess heat as opposed to electricity output.
-*' @limitations Water demand is calculated in a post-processing of REMIND and not part of the optimization.
+*' @description Water demand is calculated based on data on water demand coefficients (both for consumption 
+*' and withdrawal) and cooling shares.
+*' 
+*' For thermal power plants, water demand is proportional to excess heat production, which is calculated as fuel input minus
+*' electricity output and smokestack losses (assumed to be 10% of fuel input for non-nuclear thermal power plants).
```

Comment:
is there a reference literature behind this assumption?

Reply:
Yes, this is from Ioanna's paper: https://www.sciencedirect.com/science/article/abs/pii/S146290111630301X?via%3Dihub,
which cites this one: https://www.sciencedirect.com/science/article/abs/pii/S1462901115300745